Mention contact email verification in contact management article (#1913)
dnsimple/dnsimple-support: content/articles/contact-management.md

![creating a new contact](/files/change-contact-1.png)

> [!NOTE]

> > If the email address on the contact has not been verified, you will receive a verification email. Follow the link in the email to complete the verification.

Reviewer: ```suggestion
> If the email address on the contact has not been verified to comply with [NIS2 regulations](https://nis2directive.eu/what-is-nis2/), you will receive a verification email. Follow the link in the email to complete the verification.
```

Reviewer: I'd defer to CS on how much we want to say.

Author: @sbastn @Pemacaa @stephanoodle what's your take on this ☝️

Reviewer 2: The longer version with the email address it comes from is better. I can still see us getting "is this legit?" emails, but this should cut down on them.

Reviewer 3: Yes, the second version is better because it provides a bit more context.

Author: Applying suggestion then!

> [!NOTE]

> > When a domain contact has been changed, you will receive a confirmation email from no-reply@ispapi.net. You may also see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/).

Author: ```suggestion
> [!NOTE]
> When you update a domain contact:
- You will receive a confirmation email from no-reply@ispapi.net.
- You may also see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/).
- If you update the contact's email address you may also receive an specific email to verify the new email address.
```

Author: @sbastn @Pemacaa @stephanoodle how does that sound to you ☝️

Reviewer 2: I was so confused at first with these comments. 

I like this version the best. It has all of the information we need to highlight in it.

Reviewer 3: This is perfect. Thank you @jacegu!

Author: Applying suggestion!

> [!NOTE]
> If you update a contact's email address and it has not been verified, you will receive a verification email. Follow the link in the email to complete the verification.

> 

Author: ```suggestion
```

Removing per @sbastn's suggestion [here](https://github.com/dnsimple/dnsimple-support/pull/1913#issuecomment-4353058197) so we unify in the previous note.
